### PR TITLE
Space bar

### PR DESCRIPTION
**Pull request by dikiy4eburator**
#### Summary
Content "Add a new sidebar layout"

#### Purpose of change
add a bit variety to the default bars.

#### Testing
tested on a local build

#### Additional context
ok, now the last peace is merged.
I want to introduce you to the "spacebar".

spacebar_full

BTW. the separators are linked to the pain indikator
and gradualy change their form and colors when the player get hurt.

spacebar_bp_status
(at max pain)

have fun :)

### CODE REVIEW

**Comment by liminalRoar** [comment]
Heck yeah, this looks incredible! Great work!

**Comment by dikiy4eburator** [comment]
> Heck yeah, this looks incredible! Great work!

thx :)

**Comment by dikiy4eburator** [comment]
I wanted to mention something that caught my eye,

when i build the game localy with:
make -j8 CLANG=1 CCACHE=1 RELEASE=1 TILES=1 SOUND=1 PREFIX=~/apps/cdda-test/ install
everything works as expected.

but when i build the curses version with:
make -j8 CLANG=1 CCACHE=1 RELEASE=1 PREFIX=~/apps/cdda-test/ install

I get this error message
curses_error

could that be the reason why the curses builds fail in my commits,
or is this something on my system side?

**Comment by anothersimulacrum** [comment]
The tests failed because of https://github.com/CleverRaven/Cataclysm-DDA/runs/7230874202?check_suite_focus=true#step:16:321
not your fault.

**Comment by Inglonias** [comment]
Please remove the image from the PR. It looks like your notes, and there's no need to distribute it.

File: data/json/ui/spacebar/spacebar_body_and_health_block.json [hunk 1]
```diff
@@ -0,0 +1,390 @@
+[
+  {
+    "id": "player_health_and_stasistik_block",
+    "type": "widget",
+    "style": "layout",
+    "label": "1_player_block",
+    "width": 58,
+    "height": 17,
```
Comment (Inglonias) [inline data/json/ui/spacebar/spacebar_body_and_health_block.json:8]: height does nothing in layout widgets, and can be removed.

**Comment by Inglonias** [comment]
Other than the comments I mentioned above, this looks super neat. Being able to see my encumbrance and the temperature of each individual body part on the sidebar is a feature I didn't even know I wanted.

File: data/json/ui/spacebar/spacebar_body_and_health_block.json (hunk 1 above)
Comment (dikiy4eburator) [reply data/json/ui/spacebar/spacebar_body_and_health_block.json]: ok, all done

Comment (dikiy4eburator) [reply data/json/ui/spacebar/spacebar_body_and_health_block.json]: all done

**Comment by dikiy4eburator** [comment]
> Other than the comments I mentioned above, this looks super neat. Being able to see my encumbrance and the temperature of each individual body part on the sidebar is a feature I didn't even know I wanted.

Unfortunately, the temperature is in absolute numbers, so I had to disguise it as a graph.
is it possible to output them in C° or in F, so that I can show them directly as numbers?

**Pull request merged**